CONTRIBUTING.md

```diff
@@ -120,14 +122,237 @@ rework your branch. You should rework any relevant information from the GitHub
 
-For more information on GitHub "force push" workflows see "[Why and how to
-correctly amend GitHub pull
-requests](http://blog.adamspiers.org/2015/03/24/why-and-how-to-correctly-amend-github-pull-requests/)".
-
 Your PR can contain more than one patch. Use as many patches as necessary to
 implement the request. Each PR should only cover one topic. If you mix up
 different items in your patches or PR, they will likely need to be reworked.
 
+#### Example first PR
+
+This section will guide you through how to create your first PR. We will
+create a branch and in that branch we will change *this document*. This
+section assumes no previous `git(1)` setup or knowledge:
+
+- Basic git setup:
+
+  Install the `git` package for your distribution. Next, if you haven't
+  already done this, you need to tell `git` your name and email address so these
+  can be recorded in your commits.
+
+  ```sh
+  $ git config --global user.email "you@example.com"
+  $ git config --global user.name "Your Name"
+  ```
+
+  > **Note:** The email address you specify here must match your primary GitHub email address.
+
+- Create your own copy of the repository you want to change by clicking "fork"
+  on https://github.com/kata-containers/community.
+
+  This will create your copy under `https://github.com/${your-github-username}` *on the github server*.
+
+- [Clone your fork](https://help.github.com/articles/cloning-a-repository):
+
+  "Cloning" is a term that means to create a *local copy* of a git repository
+  that generally lives on a remote server.
+
+  ```sh
+  $ user="your-github-username"
+  $ dir="$GOPATH/src/github.com/kata-containers"
+  $ mkdir -p "$dir"
+  $ cd "$dir"
+  $ git clone https://github.com/${user}/community
+  $ cd community
+  ```
+
+  There are now *three* copies of the repository:
+
+  - The original "upstream" one (https://github.com/kata-containers/community)
+
+    You do not (and cannot) make changes to this repository. PRs are a way of
+    allowing the project developers and administrators to review your work and
+    decide if they want to add your work to the upstream branch.
+
+    > **Note:** Don't worry - even if you make a mistake,
+    > you *cannot* do any damage to this repository.
+
+  - Your fork (`https://github.com/${your-github-username}`) on GitHub
+
+    This is where you will send or upload (`git push`) your PR branches.
+
+  - Your local copy of your fork (`$GOPATH/src/github.com/kata-containers/${user}/community`)
+
+    This is where you will actually make changes to your private copy of the
+    repository.
+
+- [Setup a "git remote"](https://help.github.com/articles/configuring-a-remote-for-a-fork/):
+
+  You now have a local copy of your fork. However, you need to tell `git` where
+  your fork lives. The command below simply creates an alias (name) for the URL
+  of the fork you created above. This makes dealing with forks easier.
+
+  ```sh
+  $ git remote add github https://github.com/${user}/community
```

Comment:
Will the git clone not have set up the `origin` remote already? If this is different, we may want to note how/why here (such as ssh vs http access)

Reply:
I suggest to name the remote `fork`, to make it consistent with what's written above.